### DISCUSSION TITLE
关于必应模式下，出现”实在抱歉，我现在无法回答这个问题。”无法继续对话的问题

### DISCUSSION THREAD

**Discussion by heisenyu** [ikechan8370/chatgpt-plugin#347] [Ideas]
## 问题描述

必应模式下，出现”实在抱歉，我现在无法回答这个问题。“时，无法继续该话题聊天，有什么办法解决这一问题吗？
我注意到当出现道歉时，不会将此次的内容写入到cache.json中，但是将cache.json删除后，也就相当于#结束对话，就可以新的话题，必应是通过什么来判断当前话题的，有没有办法绕过检测。

## 可能的解决方案

说一下我的设想：
1、当出现无法回答时，将cache.json中的消息撤销一条
2、如果是通过会话id进行判断的当前话题，能不能将会话id改一下
3、如果是当前对话id和父对话id，能不能通过修改json文件中的id，但仍保持连续性
4、可不可以通过提备份cache.json文件，来保留当前会话，当遇到无法回答时，可以回溯改变话题。

我是小白，稍微看了一下源码，还有很多不明白的地方，还请大佬指教一下。

**Comment by Err0rCM** [comment]
这个问题有点太宽泛了，建议扔discussion里
